package.json

```diff
@@ -106,6 +108,8 @@
     "@types/redux-mock-store": "^0.0.13",
     "@types/showdown": "^1.7.5",
     "@types/xml2js": "^0.4.3",
+    "@pusher/chatkit-client": "^1.5.0",
```

Comment:
This dep shouldn't be dev dependency, only the "@types" should be. 

Dev dependencies are dependencies that won't appear in production. "@types" are not needed because our typescript files will be compiled to js files.